vendor/cloud.google.com/go/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Don't checkin vendor directory. Libraries/packages should only declare their vendor requires in Gopkg.toml

Reply:
OK, I removed the commit which updated the vendor tree